test/com.cooltt

```diff
@@ -20,4 +20,21 @@ def com/fun : {
   com/B r phi {\j _ => p j _ {com/A r' {1==0} {\_ _ => x} j}} r'
 }
 
-normalize com/fun
+normalize mycom/fun
+
+def coe/intro : {
+  (A : dim -> univ) (r : dim) (r' : dim) (x : A r) ->
+  sub {A r'} {r==r'} x
+} = {
+ \A r r' x => coe A r r' x
+}
+
+def coe/fun : {
+  (A : dim -> univ) (B : dim -> univ)
+  (r : dim) (r' : dim)
+  (f : (_ : A r) -> B r) ->
```

Comment:
It is currently a syntax error to write `(f : A r -> B r)` or even `(f : (A r) -> B r)`.

Reply:
Taste it, we didn't add non-dependent function notation!

Also, please note that parentheses qua grouping are braces in cooltt.

Reply:
Is `A : dim -> univ` a special case? I assumed this meant that we did have non-dependent function notation.

Reply:
Oh, maybe I'm wrong LOL. Try `(f : {A r} -> B r)` ??? 

The parser is no good lol

Reply:
(braces work, fixed in 59da607)